src/Core/Types.Tests/Utilities/XmlDocumentationExtensionTests.cs

```diff
@@ -0,0 +1,312 @@
+using System.Drawing;
```

Comment:
This file is effectively a copy-paste of [`XmlDocumentationExtensions`](https://github.com/RicoSuter/NJsonSchema/blob/master/src/NJsonSchema/Infrastructure/XmlDocumentationExtensions.cs) in NJsonSchema, and adapted to use the correct .NET Standard APIs as opposed to dynamic invocation through reflection.